src/components/SearchBar/SearchBar.spec.js

```diff
@@ -0,0 +1,22 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
+import SearchBar from './index';
+
+describe('SearchBar component', () => {
+	it('should render children', () => {
+		const handleChange = jest.fn();
+
+		render(<SearchBar searchChallenges={handleChange} searchInput="the" />);
+
+		const form = screen.getByTestId('form');
+		const svg = screen.getByTestId('img-search');
+		const searchInput = screen.getByPlaceholderText('search...');
```

Comment:
I would suggest `getByLabelText` instead as it also test for accessibility and that there actually is a `label` :)

Reply:
add a label